### PR TITLE
Removed Prosecution Concluded Listener and Scheduler

### PR DESCRIPTION
**Pull request by muthusubra**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-XXX)

Describe what you did and why.

## Checklist

Before you ask people to review this PR:

- [ ] Tests should be passing: `./gradlew test`
- [ ] Github should not be reporting conflicts; you should have recently run `git rebase main`.
- [ ] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [ ] You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- [ ] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: aws/application/application.template [hunk 1]
```diff
@@ -534,10 +528,6 @@ Resources:
               Value: !Ref pEnvironment
             - Name: POST_MVP_ENABLED
               Value: !Ref pPostMvpEnabled
-            - Name: PROSECUTION_CONCLUDED_LISTENER_ENABLED
-              Value: !Ref pProsecutionConcludedListenerEnabled
-            - Name: PROSECUTION_CONCLUDED_SCHEDULE_ENABLED
-              Value: !Ref pProsecutionConcludedScheduleEnabled
```
Comment (Mhowell494) [inline aws/application/application.template:540]: Missed reference to `LAA_PROSECUTION_CONCLUDED_QUEUE` environment variable on lines 533 - 534.

File: aws/application/application.template [hunk 2]
```diff
@@ -62,12 +62,6 @@ Parameters:
...
     Default: ''
-  pProsecutionConcludedListenerEnabled:
-    Type: String
-    Default: ''
-  pProsecutionConcludedScheduleEnabled:
-    Type: String
-    Default: ''
   pCreateLinkCpStatusJobQueue:
     Type: String
     Default: ''
```
Comment (Mhowell494) [inline aws/application/application.template:67]: Missed reference to `pLaaProsecutionConcludedQueue` environment variable on lines 68 - 70.

File: maat-court-data-api/src/testSqsIntegration/resources/application.yaml [hunk 3]
```diff
@@ -107,10 +106,6 @@ swagger-ui:
 
 feature:
   postMvpEnabled: true
-  prosecution-concluded-listener:
-    enabled: true
-  prosecution-concluded-schedule:
-    enabled: true
```
Comment (Mhowell494) [inline maat-court-data-api/src/testSqsIntegration/resources/application.yaml:113]: Need to also remove this configuration in the main `application.yaml`

**Review by Mhowell494 — CHANGES_REQUESTED** [review]
- I've noticed the prosecution concluded package hasn't been entirely deleted, is there a reason we can't do this? I've noticed there are a few helpers that are referenced from elsewhere, I think we should move these to a more appropriate location (along with any other required classes) and delete the package.

- Need to clean up Docker files as these still contain references to environment variables that have now been removed.

- There are also a number of other references, which need to be cleaned up.

File: aws/application/application.template (hunk 1 above)
Comment (muthusubra) [reply aws/application/application.template:540]: Removed

File: maat-court-data-api/src/testSqsIntegration/resources/application.yaml (hunk 3 above)
Comment (muthusubra) [reply maat-court-data-api/src/testSqsIntegration/resources/application.yaml:113]: Removed

**Review by Mhowell494 — APPROVED** [review]

**Pull request merged**